docs/included.md

```diff
@@ -105,6 +105,13 @@
   - &mdash;
   - &mdash;
   - &mdash;
+* - Quantile Interval Score (Interval Score)
+  - [API](api.md#scores.continuous.quantile_interval_score)
+  - [Tutorial](project:./tutorials/Quantile_Interval_Score.md)
+  - [Gneiting and Raftery (2007) - Corollary 5.2](https://doi.org/10.1198/016214506000001437)
+  - &mdash;
+  - &mdash;
+  - &mdash;
```

Comment:
```suggestion
```

Reply:
So if we had an `interval_score` it would have only one quantile level that implied the second quantile level and otherwise be the same as the `quantile_interval_score`? 

I suggest we don't need an `interval_score`, but it would be good if it is easy for people looking for an interval_score to find the quantile_interval_score.

Reply:
That makes sense. If the vanilla interval_score method is commonly-used / likely to be of interest, it would obviously be straightforward to add an interval_score method which called quantile_interval_score accordingly. Whether it's worth doing I couldn't say... But it might be easier to implement it than explain why it's not there...

Reply:
I agree, and I will add interval_score method that calls the quantile_interval_score function.

Reply:
Thank @Steph-Chong, I've added `interval_score` functions (as an individual function that calls `quantile_interval_score`) and corresponding docs.